content/logs/dynamic_volume_control.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Dynamic volume control
+kind: documentation
+description: 
+further_reading:
+- link: "logs/graph"
+  tag: "Documentation"
+  text: "Perform analytics with Log Graphs"
+- link: "logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+---
+
+<div class="alert alert-warning">
+This feature is currently in Beta. Ask your Sales representative or your Customer Success Manager to get it enabled.
```

Comment:
Ask your Sales representative or Customer Success Manager to enable this feature.